core-domain/src/main/java/org/uniprot/core/uniref/impl/UniRefEntryLightImpl.java

```diff
@@ -186,12 +171,10 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         UniRefEntryLightImpl that = (UniRefEntryLightImpl) o;
-        return getCommonTaxonId() == that.getCommonTaxonId()
-                && getSequenceLength() == that.getSequenceLength()
+        return getSequenceLength() == that.getSequenceLength()
                 && getMemberCount() == that.getMemberCount()
```

Comment:
I think, we should include field commonTaxon in equals and hashcode.

Reply:
we have it in line 183 and 200

Reply:
oh ok. I didnt see. sorry